andela/lotus-ah-frontend
# Ft update profile 160451560

#### What does this PR do?
A user should be able to update profile functionality

#### Description of Task to be completed?
A user can update their profile but can not update other user's profile.

#### How should this be manually tested?
Anyone with the right url can view any user profile. From localhost;

1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 5000
3. visit the right url e.g http://localhost:8080/profile/Prince_41 to see a user profile.
4. On the page click on the edit profile button
5. Edit the fields and click save button, then you are done.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[#160451560](https://www.pivotaltracker.com/story/show/160451560)

#### Screenshots (if appropriate)
<img width="1679" alt="screenshot 2018-11-29 at 12 07 30 pm" src="https://user-images.githubusercontent.com/14542449/49217977-76b2a880-f3cf-11e8-9aea-d31898464f35.png">

#### Questions:
N/A
